native/cocos/bindings/manual/jsb_global.cpp

```diff
@@ -659,7 +662,69 @@ static bool js_loadImage(se::State &s) { // NOLINT
     return false;
 }
 SE_BIND_FUNC(js_loadImage)
+//pixels(RGBA), width, height, fullFilePath(*.png/*.jpg)
+static bool js_saveImageData(se::State& s) // NOLINT
+{
+    const auto& args = s.args();
+    size_t argc = args.size();
+    CC_UNUSED bool ok = true;
```

Comment:
Don't need `CC_UNUSED `, it is used. Please not just copy from auto binding codes. Modify it manually.

Reply:
1